SS14.Admin/AdminLogs/AdminLogRepository.cs

```diff
@@ -1,100 +1,110 @@
-﻿using System.Globalization;
-using System.Text.RegularExpressions;
+﻿using System.Data;
+using System.Globalization;
+using System.Text.Json;
 using Content.Server.Database;
 using Content.Shared.Database;
+using Dapper;
 using Microsoft.EntityFrameworkCore;
+using SS14.Admin.Models;
 using SS14.Admin.Pages.Logs;
 
 namespace SS14.Admin.AdminLogs;
 
 public static class AdminLogRepository
 {
-    private static readonly Regex ParameterRegex = new(Regex.Escape("#"));
 
-    public static async Task<List<AdminLog>> FindAdminLogs(
-        ServerDbContext context,
-        DbSet<AdminLog> adminLogs,
-        string? playerUserId,
-        DateTime? fromDate, DateTime? toDate,
-        string? serverName,
-        LogType? type,
-        string? search,
-        int? roundId,
-        int? severity,
-        LogsIndexModel.OrderColumn sort,
-        int limit = 100, int offset = 0)
+    public class WebAdminLog
     {
-        var fromDateValue = fromDate?.ToString("o", CultureInfo.InvariantCulture);
-        var toDateValue = toDate?.AddHours(23).ToString("o", CultureInfo.InvariantCulture);
-        var typeInt = type.HasValue ? Convert.ToInt32(type).ToString() : null;
-
-        var values = new List<object>();
-        if (fromDateValue != null && toDateValue != null)
-        {
-            values.Add(fromDateValue);
-            values.Add(toDateValue);
-        }
+        public int RoundId { get; set; }
+        public int Id { get; set; }
+        //not used but leave it incase
+        public Round Round { get; set; } = default!;
+        public LogType Type { get; set; }
+        public LogImpact Impact { get; set; }
+        public DateTime Date { get; set; }
+        public string Message { get; set; } = default!;
+        public JsonDocument Json { get; set; } = default!;

```

Comment:
I think you're still not disposing of the JsonDocument

Reply:
i forgor